### DISCUSSION TITLE
Swap Tab and Right Arrow key bindings?

### DISCUSSION THREAD

**Discussion by cosmoKenney** [chrisant996/clink#337] [General]
Looking at the docs here: https://chrisant996.github.io/clink/clink.html#customizing-key-bindings
It appears that I should be able to swap the behavior of Tab and Right Arrow.  Is there any caveat to just assigning Tab to the current Right Arrow behavior? And vice versa?

Reason I want to do that is I am used so used to how completion works in Visual Studio and SSMS that I cannot get myself to use right arrow to complete the suggested value. FYI, I'm on windows 11. Installed clink-maintained via Chocolatey. And I think that turned on "windows" defaults for completion.

**Comment by chrisant996** [comment]
The only caveats are the obvious ones:
- If other people use the computer, the swapped keys might confuse them.
- When reading documentation and posts that refer to the keys, you'll have to translate the keys according to your re-mappings.
- When writing posts and referring to any of the re-mapped key, mentioning which of your keys are remapped with help readers understand the post.

**Reply by cosmoKenney** [reply]
Thanks. I'm starting to realize that since right arrow is dual-purpose, this might not be a good idea. Since mapping tab to complete auto suggestion, then it would also change the cursor movement while editing a command line.
